client/src/pages/flavor_page/FlavorPage.tsx

```diff
@@ -0,0 +1,64 @@
+import styles from ".flavorpage.module.css";
+import React from "react";
+import ToppingsBar from "../../Components/ToppingsBar";
+import Suggestions from "../../Components/Suggestions";
+import ProductCard from "../../Components/ProductCard";
+import { useParams, useNavigate } from "react-router-dom";
+import { useEffect, useState, useRef } from "react";
+import axios from "axios";
+
+const flavorSuggestions = await axios
+  .get(`http://localhost:3003/api/icecreams/suggestions/3`)
+  .then((res) => res.data);
```

Comment:
are you meaning to use await with a .then?

Reply:
I've only marked this one, but there's like 6 more if this isn't an intentional choice

Reply:
removed .then in FlavorPage and substituted with async await

Reply:
Great!